### PR TITLE
updated azure

### PR DESCRIPTION
**Pull request by Anjugeorg-e**

### CODE REVIEW

File: AzureOriginal/index.html [hunk 1]
```diff
@@ -0,0 +1,504 @@
...
+        <div class="space"> </div>
+      </div>
+      <div class="azure_infra boxz">
+        <img src="Asset/img/discover-card5.avif" alt="">
+        <div class="discover_five box">
+          <h3>AZURE GLOBAL INFRASTRUCTURE</h3>
+          <p>Go beyond the limits of on-premises datacenters with more regions than any other provider.</p>
+          <a href="#">Learn more ></a>
+        </div>
+        <div class="space"> </div>
```
Comment (APJ-9) [inline AzureOriginal/index.html:315]: why there is a space class here?

File: AzureOriginal/style.css [hunk 2]
```diff
@@ -0,0 +1,1829 @@
...
+        color: #1A1A1F;
+        font-family: "cereal";
+    }
+    .footertwo_nav ul {
+        margin: 12px 0 0;
+    }
+    .footertwo_nav li {
+        font-size: 12px;
+        color: #1A1A1F;
+        padding: 2px 0;
```
Comment (APJ-9) [inline AzureOriginal/style.css:1378]: group similar styles

File: AzureOriginal/style.css [hunk 3]
```diff
@@ -0,0 +1,1829 @@
...
+        margin: 0;
+        padding: 54px;
+        background-image: url(Asset/img/Desktop\ hero.jpeg);
+        background-repeat: no-repeat;
+        background-position: center;
+        height: fit-content;
+        background-size: cover;
+        width: 100%;
+        height: 100%;
+        padding-bottom: 42px;
```
Comment (APJ-9) [inline AzureOriginal/style.css:1499]: Why 2 padding here?

File: AzureOriginal/style.css [hunk 4]
```diff
@@ -0,0 +1,1829 @@
...
+    .cards {
+        padding-left: 4%;
+        padding-right: 4%;
+    }
+    .tab-panel {
+        position: relative;
+    }
+    .tab-panel-container {
+        padding: 0 12px;
+        padding-left: 5%;
```
Comment (APJ-9) [inline AzureOriginal/style.css:1561]: Why 2 kinds of padding here?

File: AzureOriginal/style.css [hunk 5]
```diff
@@ -0,0 +1,1829 @@
...
+        -webkit-text-size-adjust: 100%;
+        width: 473px;
+        padding-top: 0px;
+    }
+    #mainone h1 {
+        margin-top: -9px;
+    }
+    #mainone h2 {
+        margin: 0 0 20px;
+        margin-top: 3px;
```
Comment (APJ-9) [inline AzureOriginal/style.css:1514]: Why 2 kinds of margins here?

File: AzureOriginal/style.css [hunk 6]
```diff
@@ -0,0 +1,1829 @@
...
+    }
+    .azure_footer {
+        margin-top: 36px;
+    }
+}
+
+@media screen and (min-width: 1024px) {
+    .tab-links a:hover {
+        background-color: #dcdcdc;
+        border: 0px solid transparent;
```
Comment (APJ-9) [inline AzureOriginal/style.css:1582]: what is the purpose of this code `border: 0px solid transparent;`

File: AzureOriginal/style.css [hunk 7]
```diff
@@ -0,0 +1,1829 @@
...
+        margin: 0 0 24px;
+    }
+    .video_azure {
+        position: relative;
+    }
+    .video_game {
+        position: absolute;
+        padding-left: 5%;
+        padding-right: 2%;
+        margin-left: 5%;
```
Comment (APJ-9) [inline AzureOriginal/style.css:1606]: group similar styles

**Review by APJ-9 — CHANGES_REQUESTED** [review]

**Review by APJ-9 — COMMENTED** [review]
Add screen shots of all the view

File: AzureOriginal/index.html (hunk 1 above)
Comment (Anjugeorg-e) [reply AzureOriginal/index.html:315]: renamed to hybrid_box now

File: AzureOriginal/style.css (hunk 2 above)
Comment (Anjugeorg-e) [reply AzureOriginal/style.css:1378]: updated

File: AzureOriginal/style.css (hunk 3 above)
Comment (Anjugeorg-e) [reply AzureOriginal/style.css:1499]: updated to a single padding value

File: AzureOriginal/style.css (hunk 4 above)
Comment (Anjugeorg-e) [reply AzureOriginal/style.css:1561]: changed

File: AzureOriginal/style.css (hunk 5 above)
Comment (Anjugeorg-e) [reply AzureOriginal/style.css:1514]: updated to single margin value

File: AzureOriginal/style.css (hunk 6 above)
Comment (Anjugeorg-e) [reply AzureOriginal/style.css:1582]: removed now. i thought of adding a blue border in previous pixel once we hover over  that . now cleared all.

**Comment by Anjugeorg-e** [comment]
azure_mobileview
azure_tablet
azure_laptop

**Review by APJ-9 — APPROVED** [review]

**Pull request merged**